public/app/features/dashboard-scene/scene/NavToolbarActions.tsx

```diff
@@ -221,8 +221,8 @@ export function ToolbarActions({ dashboard }: Props) {
     ),
   });
 
-  if (dynamicDashNavActions.left.length > 0 && !isEditingPanel) {
-    dynamicDashNavActions.left.map((action, index) => {
+  if (dynamicDashNavActions.right.length > 0 && !isEditingPanel) {
+    dynamicDashNavActions.right.map((action, index) => {
```

Comment:
Should this be here?

Reply:
ah, no this was just to test a drawer using ModalsContext (and its opened by a button in here, but this is already fixed in a separate PR)